stash_datacite/app/views/stash_datacite/publications/update.js.erb

```diff
@@ -0,0 +1,12 @@
+// switch do_import hidden value back to false in case form needs to be played with some more
+$('#internal_datum_do_import').val('false');
+
+<% if @results.code > 199 && @results.code < 300 %>
+  // get data into DB somehow and reload the page to show the pretty new data, which also resets this form to defaults
+  location.reload(true);
```

Comment:
Why not just re-render the partial here instead of using JS location? I think it would be more Rails-ish

Reply:
Because there are about 15 forms on the page and we'd have to re-render all of them.  Definitely could be done but much simpler and more trustworthy to reload the page and be sure all the forms and also geolocation, mapping and everything else gets loaded correctly.